.gitignore

```diff
@@ -4,7 +4,7 @@ npm-debug.log
 yarn.lock
 .vscode
 package-lock.json
-
+docsRNC/
```

Comment:
You probably meant to add this to `.npmignore`

Reply:
correct